code/modules/mob/living/carbon/emote.dm

```diff
@@ -279,32 +279,65 @@
 
 /datum/emote/living/carbon/sound/run_emote(mob/user, params)
 	var/mob/living/carbon/human/H = user
-	if (!istype(H))
+	if(!istype(H))
 		return ..()
-	if (H.stat == DEAD)
+	if(H.stat == DEAD)
 		return
 	if (!H.is_muzzled() && !issilent(H)) // Silent = mime, mute species.
-		if (params == TRUE) // Forced scream
+		if(params == TRUE) // Forced scream
 			if(world.time-H.last_emote_sound >= 30)//prevent scream spam with things like poly spray
 				if(sound_message)
 					message = sound_message
+				var/obj/item/clothing/C = search_sound_clothing(H)
 				var/sound
-				if (isvox(H) || isskelevox(H))
-					sound = pick(birb_sounds)
-
-				else
-					if(Holiday == APRIL_FOOLS_DAY && H.mind.assigned_role in science_positions && H.gender == MALE)
-						sound = pick(science_sounds) //ACK
+				if(!C || !(key in C.sound_change))
+					if(isvox(H) || isskelevox(H))
+						sound = pick(birb_sounds)
 					else
 						switch(H.gender)
-							if (MALE)
+							if(MALE)
 								sound = pick(male_sounds)//AUUUUHHHHHHHHOOOHOOHOOHOOOOIIIIEEEEEE
-							if (FEMALE)
+							if(FEMALE)
 								sound = pick(female_sounds)
+				else
+					sound = pick(C.sound_file)
 				playsound(user, sound, 50, 0)
 				H.last_emote_sound = world.time
 
 	else
 		message = "makes a very loud noise."
 
 	return ..()
+
+//A lengthy checks that returns the clothes to be used by other procs
+/datum/emote/living/carbon/sound/proc/search_sound_clothing(mob/living/carbon/human/user)
+	var/selected_clothing //Check the clothing we've selected to be played
+	var/list/priority_high
```

Comment:
Since you're initializing these to null, the first time a `+=` is ran it'll actually do a `=`, meaning these priority lists will be the same instance as the list on the item.

Then if you're wearing multiple pieces of clothing that modify that sound priority, *another* `+=` will get ran, which'll mutate the first clothing item's list.